src/main/resources/static/javascript/mainApp/admin.controller.js

```diff
@@ -469,6 +472,8 @@
          * @param groupingPath - The path of the grouping to display owners
          */
         $scope.getGroupingOwnersOnClick = (groupingPath) => {
+            if ($scope.loadingOwners === true) {return;} // Prevent multiple triggers while loading modal
```

Comment:
Sorry, one more thing if you could expand the brackets like this:
```
// Prevent multiple triggers while loading modal
if ($scope.loadingOwners === true) {
    return;
}
```
 Turns out Codacy gave a terrible suggestion lol

Reply:
Okay got it, thank you